packages/console/src/assets/docs/guides/web-java-spring-boot/index.ts

```diff
@@ -3,10 +3,13 @@ import { ApplicationType } from '@logto/schemas';
 import { type GuideMetadata } from '../types';
 
 const metadata: Readonly<GuideMetadata> = Object.freeze({
-  name: 'Java Web',
-  description:
-    'Java Web is a web framework for Java that enables developers to build secure, fast, and scalable server applications with the Java programming language.',
+  name: 'Java Spring Boot Web',
+  description: 'Integrate Logto with a Java Spring Boot web application.',
```

Comment:
no need to change the description?

Reply:
yes, i mean this description should be relevant to the Spring Boot framework

Reply:
updated